R/sensorreading-data.R

```diff
@@ -47,7 +47,7 @@ formatSensorData <- function(data, columnNames){
     listElements <- data[listRows,]
     
     if ("displayTime" %in% names(data)) {
-      if(!is.na(listElements$displayTime) || is.null(listElements$time)) {
+      if(!isEmpty(listElements$displayTime) || isEmpty(listElements$time)) {
```

Comment:
@mhines-usgs @thongsav-usgs @dpattermann-usgs what are your thoughts on my previous comment? Should we be concerned?